README.md

```diff
@@ -0,0 +1,123 @@
+# CMIS 1.1 compliant opencmis server
+
+## Docker
+```
+docker build -t opencmis .
+docker run -p 9000:8080 opencmis
+```
+
+## How to use it
+### CMIS 1.1
+WS (SOAP) Binding: http://localhost:9000/cmis/services11/cmis?wsdl
+AtomPub Binding: http://localhost:9000/cmis/atom11
+Browser Binding: http://localhost:9000/cmis/browser
+
+### CMIS 1.0
+WS (SOAP) Binding: http://localhost:9000/cmis/services/cmis?wsdl
+AtomPub Binding: http://localhost:9000/cmis/atom
+
+### Authentication
+Basic Authentication 
+
+Default users :
+  * user1 / cm1sp@ssword
+  * user2 / cm1sp@ssword
+  * user3 / cm1sp@ssword
+  * user4 / cm1sp@ssword
+  * user5 / cm1sp@ssword
+
+
+# Basic CMIS Docker image <!-- omit in toc -->
+
+![Docker Stars](https://img.shields.io/docker/stars/exoplatform/exo.svg) - ![Docker Pulls](https://img.shields.io/docker/pulls/exoplatform/exo.svg)
+
+- [CMIS 1.1 compliant opencmis server](#cmis-11-compliant-opencmis-server)
+  - [Docker](#docker)
+  - [How to use it](#how-to-use-it)
+    - [CMIS 1.1](#cmis-11)
+    - [CMIS 1.0](#cmis-10)
+    - [Authentication](#authentication)
+  - [Configuration options](#configuration-options)
+    - [CMIS](#cmis)
+    - [JVM](#jvm)
+    - [Frontend proxy](#frontend-proxy)
+    - [Tomcat](#tomcat)
+    - [JMX](#jmx)
+
+## Configuration options
+
+All the following options can be defined with standard Docker `-e` parameter
+
+```bash
+docker run -e MY_ENV_VARIABLE="value" ... exoplatform/cmis-server
+```
+
+or Docker Compose way of defining environment variables
+
+```yaml
+version: '2'
+services:
+...
+  exo:
+    image: exoplatform/cmis-server
+    environment:
+...
+      JVM_LOG_GC_ENABLED: true
+      CMIS_USERS_PASSWORD: mycomplicatedpassword
+...
+```
+
+### CMIS
+
+5 users (``user1``, ..., ``user5``) are created by default in the cmis container. Their password can be provided with this environment option :
+
+| VARIABLE               | MANDATORY | DEFAULT VALUE   | DESCRIPTION                                               |
```

Comment:
All your table are not well formatted and contains extra spaces

use `fcrespo82.markdown-table-formatter` add-on in Visual-Source code with following settings

`
    "markdown.extension.tableFormatter.normalizeIndentation": true,
`

Reply:
👍 Updated